openapi/listings/listings.yaml

```
@@ -82,25 +82,25 @@ paths:
                         description: The URI at which the next batch of reviews can be gotten from
                         format: uri
     options:
-      operationId: options-businessEntityListings
-      summary: List valid HTTP verbs for /businessEntityListings
+      operationId: options-listingSyncListings
+      summary: List valid HTTP verbs for /listingSyncListings
       description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
       responses:
         '204':
           description: No Content
         '403':
-          description: User Cannot Access Entity
+          description: User Cannot Access Business Location
         '404':
           description: Business Not Found
         '406':
           description: No Listing Product Active
       tags:
         - Options
-        - Business Entity Listings
+        - Listing Sync Listings
     parameters: []
-  '/businessEntityListingScores/{id}':
+  '/listingScore/{id}':
```

This should be plural.
```suggestion
  '/listingScores/{id}':
```

---

Should the score be prefixed with Listing Sync to match the listings? 

```suggestion
  '/listingSyncScores/{id}':
```

---

Not according to the meeting minutes https://vendasta.jira.com/browse/LB-2347